pkg/alert/ctl_alert.go

```diff
@@ -167,8 +152,9 @@ func (ctl *Ctl) SetFlag(f *pflag.Flag) {
 		case "cfssl-image":
 			ctl.Spec.CfsslImage = ctl.CfsslImage
 		case "stand-alone":
-			ctl.Spec.StandAlone = &ctl.StandAlone
-		case "expose-service":
+			standAloneVal := strings.ToUpper(ctl.StandAlone) == "TRUE"
```

Comment:
@mphammer I wonder if use of [`strconv.ParseBool`](https://golang.org/pkg/strconv/#ParseBool) would be better here.